keps/sig-network/1672-tracking-terminating-endpoints/README.md

```diff
@@ -129,8 +130,11 @@ endpoints is correct.
 
 ### Graduation Criteria
 
-Since this is an addition to the EndpointSlice API, graduation will follow the graduation
-timeline for the [EndpointSlice API work](/keps/sig-network/20190603-endpointslices/README.md).
+#### Alpha
+
+* EndpointSlice API includes `Terminating` condition.
+* `Terminating` condition can only be set if feature gate `EndpointSliceTerminatingCondition` is enabled.
+* Unit tests in endpointslice controller and API validation/strategy.
```

Comment:
Can you add a bit more detail here around what these would cover?

Reply:
This is covered under the "Test Plan" section https://github.com/kubernetes/enhancements/blob/0af741e352e8f498a8aeda0fa423b66ca09de836/keps/sig-network/1672-tracking-terminating-endpoints/README.md#test-plan 

I admit though that what's written in the KEP today is off from the tests I ended up writing in https://github.com/kubernetes/kubernetes/pull/92968, will update that now